Kentor.AuthServices.Tests/Saml2P/Saml2AuthenticationRequestTests.cs

```diff
@@ -416,5 +428,16 @@ private void Saml2AuthenticationRequest_ToXElement_AddsRequestedAuthnContextUtil
 
             actual.Should().BeEquivalentTo(expected);
         }
+
+        private void Saml2AuthenticationRequest_ToXElement_AddsProtocolBinding(AuthServices.WebSso.Saml2BindingType protocolBinding, string expectedProtocolBinding)
+        {
+            var subject = new Saml2AuthenticationRequest()
+            {
+                AssertionConsumerServiceUrl = new Uri("http://destination.example.com"),
+                Binding = protocolBinding
+            }.ToXElement();
+
+            subject.Attribute("ProtocolBinding").Value.Should().Equals(expectedProtocolBinding);

```

Comment:
In fluent assertions you should use `.Be` and not `.Equals`. The former throws an exception if they are not equal, the latter returns a value that is ignored. I'll fix when merging.